catroid/src/main/java/org/catrobat/catroid/content/actions/ThinkSayBubbleAction.java

```diff
@@ -47,14 +47,15 @@ protected void update(float delta) {
 			Log.d(getClass().getSimpleName(), "Formula interpretation for this specific Brick failed.", interpretationException);
 			return;
 		}
-		ShowBubbleActor showBubbleActor = new ShowBubbleActor(textToDisplay, sprite, type);
-
 		if (StageActivity.stageListener.getBubbleActorForSprite(sprite) != null) {
```

Comment:
If you have a program:

ShowBubble "Hello"
Wait 10 Seconds
ShowBubble ""

The Bubble with "Hello" would disappear, is this intended ?